test/test_lexer.py

```diff
@@ -1006,6 +1006,213 @@ def _dont_test_dict_expression_2_issue_400(self):
             ),
         )
 
+    def test_dict_expression(self):
```

Comment:
can you uncomment the "_dont_test_dict_expression_issue_400" and "_dont_test_dict_expression_2_issue_400"  tests also?  these were the original tests